metatron/src/main/java/uk/gov/ida/eidas/metatron/domain/EidasConfig.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.ida.eidas.metatron.domain;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.function.Function;
+
+public class EidasConfig {
+    @JsonProperty
```

Comment:
What's this annotation for? I've seen it being used like this but wasn't sure what using it without an argument did?

Reply:
I see it as a marker annotation without any arguments; as in making it clear that this file comes from some json at some point and that this is the 'thing' that gets populated by the json.  So it's an annotation in the simplest sense of the word.

Reply:
I thought I'd replied to this earlier but it seems not.  I take it as being an annotation in the simplest sense of the word - it just tells the casual observer that this is going to be populated by some json.  It will work without it but I feel like it's useful information to have in place.